.github/workflows/go.yaml

```diff
@@ -25,6 +25,8 @@ jobs:
       fail-fast: false
       matrix:
         include:
+          - tag: "1.22"
```

Comment:
We don't want to have non-alpine tags for now. Would you mind removing that one from the matrix, please?

Reply:
ok done!